Phraseanet-production-client/src/components/record/recordEditor/index.js

```diff
@@ -213,6 +215,11 @@ const recordEditorService = services => {
                     $editDateArea.css('width',210);
                 }
             })
+            .on('mouseup mousedown keyup keydown', '.editor-thesaurus-search', (event) => {
```

Comment:
maybe only on keyup ? to be user-tested to see if refresh on click is expected or annoying...